### PR TITLE
WIP: add zh-cn version

### PR DESCRIPTION
**Pull request by magaofei**
add zh-cn version

### CODE REVIEW

**Comment by ajinabraham** [comment]
@magaofei Is this ready?

**Comment by magaofei** [comment]
> @magaofei Is this ready?

Not ready, please wait a couple of hours.

**Comment by magaofei** [comment]
@ajinabraham It‘s ready for Chinese version. But I don't know how to preview in my computer.

**Comment by ajinabraham** [comment]
@magaofei Quite easy
```
npm i docsify-cli -g
docsify serve docs
```

File: zh-cn/_navbar.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- _navbar.md -->
+<!-- docs/_sidebar.md -->
+* 入门
+    * [入门](/)
+    * [要求](requirements.md)
+    * [安装]](installation.md)
```
Comment (ajinabraham) [inline zh-cn/_navbar.md:6]: remove extra `]`

**Comment by magaofei** [comment]
it's ready for publish. @ajinabraham

**Comment by ajinabraham** [comment]
Thanks @magaofei

**Pull request merged**